test/integration/integration.cc

```diff
@@ -212,8 +211,8 @@ void IntegrationTcpClient::ConnectionCallbacks::onEvent(uint32_t events) {
 
 BaseIntegrationTest::BaseIntegrationTest()
     : api_(new Api::Impl(std::chrono::milliseconds(10000))),
-      dispatcher_(api_->allocateDispatcher()) {
-
+      dispatcher_(api_->allocateDispatcher()),
+      default_log_level_(TestEnvironment::getOptions().logLevel()) {
```

Comment:
I'm a little confused why this whole getOptions() dance is needed (referring to all of the code in this PR that sets up the static). I spend a few minutes looking, but can you maybe just summarize?